server/portal/apps/webhooks/views.py

```diff
@@ -214,7 +195,7 @@ def post(self, request, *args, **kwargs):
 
             return HttpResponse('OK')
 
-        except (ObjectDoesNotExist, AgaveException, PortalLibException) as e:
```

Comment:
Might we still want to catch the Tapis v3 equivalent of exceptions here? We've been using `BaseTapyException`

Reply:
Right, I was unsure of what the v3 equivalent of that was. I'll add that in!